templates/registration/login.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form method="post" action="{% url 'taxi:login' %}">
```

Comment:
The URL name `taxi:login` may not be correct if you are using Django's built-in authentication URLs. Typically, the login URL is `login` without the app namespace. Ensure that the URL name matches the one defined in your URL configuration.